### PR TITLE
Added simple payment

### PR DESCRIPTION
**Pull request by samigeorgiev**

### CODE REVIEW

File: client/src/components/ConfirmationModalBody.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                <Dimmer active>
+                    <Loader />
+                </Dimmer>
+            ) : (
+                props.children
+            )}
+        </Modal.Content>
+        <Modal.Actions>
+            <Button
+                onClick={props.onClose}
```
Comment (toniyanchev) [inline client/src/components/ConfirmationModalBody.jsx:19]: better use space before and after '='

Comment (toniyanchev) [inline client/src/components/ConfirmationModalBody.jsx:19]: or the eslinter force you to not?

Comment (samigeorgiev) [reply client/src/components/ConfirmationModalBody.jsx:19]: in jsx i use html code styling

File: client/src/hooks/accountOperations.js [hunk 2]
```diff
@@ -39,8 +37,8 @@ const useAccountOperations = () => {
     );
 
     return {
-        methods: { becomeTeacher, getAccount, getCourses, enrollCourse },
-        state
+        state,
```
Comment (toniyanchev) [inline client/src/hooks/accountOperations.js:40]: why is important for the state to be before methods

File: client/src/components/ConfirmationModalBody.jsx (hunk 1 above)
Comment (toniyanchev) [reply client/src/components/ConfirmationModalBody.jsx:19]: that makes sense

File: client/src/pages/ExploreCourses.jsx [hunk 3]
```diff
@@ -1,91 +1,137 @@
...
         const request = new GetCoursesByCategoryRequest();
         request.setId(category);
         getCoursesByCategory(request);
     }, [getCoursesByCategory, location]);
 
+    const { response: contentManagementResponse } = contentManagementState;
     useEffect(() => {
-        if (response) {
-            setCourses(response.getCoursesList());
+        if (contentManagementResponse) {
```
Comment (toniyanchev) [inline client/src/pages/ExploreCourses.jsx:48]: is there are difference or just code styling?

File: client/src/pages/ExploreCourses.jsx [hunk 4]
```diff
@@ -1,91 +1,137 @@
...
+                <ConfirmationModalBody
+                    title="Buy a course"
+                    isLoading={accountOperationsState.isLoading}
+                    onClose={() => setIsModalShown(false)}
+                    onConfirm={enrollHandler}
+                >
+                    {modalContent}
+                </ConfirmationModalBody>
+            </Modal>
+            <Container style={{ maxWidth: '80rem', margin: '2rem auto' }}>
```
Comment (toniyanchev) [inline client/src/pages/ExploreCourses.jsx:98]: is it better way than each attribute on a new line?

File: microservices/account-operations/src/main/java/edu/gacademy/accountoperations/clients/AuthenticationClient.java [hunk 5]
```diff
@@ -34,12 +34,17 @@ public int getUserId(String token) throws StatusRuntimeException {
     }
 
     private StatusRuntimeException wrapException(StatusRuntimeException e) {
+        if (e.getStatus() != Status.UNKNOWN) {
+            return e;
+        }
         String[] splitMessage = e.getMessage().split(" ", 3);
-        int statusCode;
+        int statusCode = Status.UNKNOWN.getCode().value();
         if (splitMessage.length >= 2) {
```
Comment (toniyanchev) [inline microservices/account-operations/src/main/java/edu/gacademy/accountoperations/clients/AuthenticationClient.java:42]: can you explain me this fragment of the code

File: client/src/hooks/accountOperations.js (hunk 2 above)
Comment (samigeorgiev) [reply client/src/hooks/accountOperations.js:40]: it is not. I just do all imports in the same style

File: client/src/pages/ExploreCourses.jsx (hunk 3 above)
Comment (samigeorgiev) [reply client/src/pages/ExploreCourses.jsx:48]: This file uses two types of responses. I just rename it for better understanding

File: client/src/pages/ExploreCourses.jsx (hunk 4 above)
Comment (samigeorgiev) [reply client/src/pages/ExploreCourses.jsx:98]: The line is under 80 chars so I prefer to keep it compact

File: microservices/account-operations/src/main/java/edu/gacademy/accountoperations/clients/AuthenticationClient.java (hunk 5 above)
Comment (samigeorgiev) [reply microservices/account-operations/src/main/java/edu/gacademy/accountoperations/clients/AuthenticationClient.java:42]: The problem is that grpc for node has extremely bad api for handling errors. That's why I send all of the error information in error message and after recieving it the client parse it to appropriate object

**Pull request merged**